# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libusb/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libusb-0.1.11/linux.c.orig	2006-02-14 22:08:34.000000000 +0100
+++ libusb-0.1.11/linux.c	2006-02-14 22:09:35.000000000 +0100
@@ -564,8 +564,8 @@
     dev->children = malloc(sizeof(struct usb_device *) * dev->num_children);
     if (!dev->children) {
       if (usb_debug > 1)
-        fprintf(stderr, "error allocating %d bytes memory for dev->children\n",
-                sizeof(struct usb_device *) * dev->num_children);
+        fprintf(stderr, "error allocating %lu bytes memory for dev->children\n",
+                (unsigned long) sizeof(struct usb_device *) * dev->num_children);
 
       dev->num_children = 0;
       close(fd);
